packages/documentation/docs/controls/_forms_custom_field_styleguide.md

```diff
@@ -0,0 +1,43 @@
+The custom field is a wrapper component that can host any other forms component. It provides a set of properties that allow you to control the validation state of the field, as well as the helper text that is displayed below the field. The custom field is a versatile tool to create your own form fields, that can be used in combination with the 'Form' components to create complex forms.
+
+![Custom field](https://www.figma.com/design/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?node-id=3303-3291&t=DlxXBQ9vTnyDcIUI-4 )
+
+1. Label
+2. Helper text/Feedback text
+3. Form components
+4. Required indicator
+%%
+## Quick guide
+
+![Custom field_new](https://www.figma.com/design/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?node-id=3482-7078&t=DlxXBQ9vTnyDcIUI-4)
+
+- **Usage:** Use the custom field when your desired solution is not covered by the standard form field component.
+- **Customize as Needed:** Add specific form components (e.g., input fields, buttons) to meet your use case.
+- **Helper and feedback text:** The custom field provides the option to display helper and feedback text automatically.
+- **Follow Guidelines:** Refer to the form field and validation chapters for detailed guidelines on labels, helper text, and feedback text.
```

Comment:
it is not a bad idea, though I think it sounds a bit mixed up from my perspective